### ISSUE TITLE
About ground truth 3d face shape 

### ISSUE THREAD

**Issue by qiexing** [anhttran/3dmm_cnn#6]
It's very kind of you to share the model, I'am inspired by your work.
 I'am a new guy to 3d face construction and I want to reproduce your paper. 
I found your paper generate ground truth 3d face shape by paper: "Estimating 3D Shape and Texture Using Pixel Intensity, Edges, Specular Highlights, Texture Constraints and a Prior". Are you implement the method? Or where can i found this code to generate ground truth 3d face shape? 
Thanks!!

**Comment by TalHassner** [comment]
@qiexing Thanks for your interest in our work. 

Though we were inspired by that paper, we actually used a simpler method to produce the training data labels, and we describe this method in our paper. 

Basically, we estimate 3DMM parameters using our own implementation (which I don't think we'll be sharing, but I believe any standard 3DMM fitting method would do the trick here). We then pool these parameter vectors (element wise average) over multiple estimates from multiple images. 

That's pretty much it. If you need a lot of 3DMM parameters for a lot of images, and don't want to implement their method, use CNN for this purpose J

I hope this helps.

**Comment by qiexing** [comment]
@TalHassner Your reply is really helper. I want to more 3d parameter information, like expression parameter. Will you release your training data? If not, do you know some code which can produce 3d parameter by a given 2d images?

**Comment by leeeeeeo** [comment]
@qiexing Same question. Have you found any code to produce 3d parameter by given 2d images?

**Comment by lyyiangang** [comment]
same question.  @leeeeeeo  I find an repo named **eos** on github , it can generate 3dmm parameter from one image. not sure it's helpful for you.
